plugin/evm/vm.go

```diff
@@ -531,6 +532,29 @@ func parseGenesis(ctx *snow.Context, genesisBytes []byte, upgradeBytes []byte, a
 	// Set network upgrade defaults
 	configExtra.SetDefaults(ctx.NetworkUpgrades)
 
+	// If Durango is scheduled, schedule the Warp Precompile at the same time
+	// (unless it's already been configured manually).
+	if configExtra.DurangoTimestamp != nil {
```

Comment:
Why we have added these? I don't think they're related with the PR or correct in general.

Reply:
I can remove it and you can see everything that fails without it - gotcha regarding though! 

> Because warp was never intended to be a required/default precompile for subnet-evm. We cannot really retroactively activate this with Durango even if we wanted it to be auto-enabled.